### DISCUSSION TITLE
Could not resolve com.trustwallet:wallet-core:2.6.23.

### DISCUSSION THREAD

**Discussion by FZ985** [trustwallet/wallet-core#1782] [Q&A]
How to obtain a user name and password when running android project

![微信图片_20211116175214](https://user-images.githubusercontent.com/16433919/141963466-00e656e9-6c75-43b1-ab0e-fc44784feee1.png)

**Comment by optout21** [comment]
Should use personal GitHub account credentials.

**Reply by ElyDantas** [reply]
I'm still getting "Username must not be null" from android sample code
Using sample url = uri("https://maven.pkg.github.com/trustwallet/wallet-core")
Got my token from "GitHub > Settings > Developer Settings > Personal access tokens > Generate new token"
Tried: "The token needs at least the read:packages scope. Otherwise you will get a Not authorized exception." from [github-plugin-registry-example](https://github.com/TobseF/github-plugin-registry-example)
Tried: using "ElyDantas", "elydantas", "elydantas.andrade@gmail.com" as user
Tried: "Change gpr.user => gpr.usr" from [this stackoverflow question](https://stackoverflow.com/questions/63881149/initially-publishing-a-github-package-shows-username-must-not-be-null)
Tried: updating to walletcore_version = "2.6.35"

**Reply by hewigovens** [reply]
@ElyDantas have you changed the settings in your `build.gradle`? you either need to add `gpr.user` and `gpr.key` to your `local.properties` or add `GITHUB_USER` and `GITHUB_TOKEN` to system env

**Reply by ElyDantas** [reply]
Thank you @hewigovens, I was missing setting `GITHUB_USER` and `GITHUB_TOKEN` as system environment variable, instead I was trying to hardcode it `System.getenv("ElyDantas")` 🤦🏻‍♂️
